dlhub_sdk/tests/test_dlhub_client.py

```diff
@@ -8,7 +8,7 @@
 class TestClient(TestCase):
 
     def setUp(self):
-        self.dl = DLHubClient.login(http_timeout=10)
```

Comment:
I figured out the problem with the tests. It has to do with this change here. Note that in `login` we do not specify a Client ID and in `__init__` we use the DLHub client ID. 

I used `login` when generating the credential files for the tests, not `__init__`. So, I'm thinking I need to get rid of the `login` function so we don't have two different Auth codes to maintain.